README.md

```diff
@@ -6,684 +6,46 @@
+### 2.2 Contributor License
 
-* What permissions or entitlements are required for the SDK?   
-  
-  Mobile Center SDK requires no permissions to be set in your app.
-  
-* The Alert that prompts users for an update doesn't contain strings, but just the keys for them?
-  This means that the `MobileCenterDistributeResources.bundle` wasn't added to the project. Make sure you have drag'n'dropped the file into your xcode project, and it appears in your app target's `Copy Bundle Resources` build phase. The later should be the case if you have added the file through drag'n'drop – Xcode does it automatically for you. If the file is missing from the build phase, add it so it get's compiled into your app's bundle. 
-  
-* Engage with other MobileCenter users and developers on [StackOverflow](http://stackoverflow.com/questions/tagged/mobile-center).
+You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
 
-## 9. Contributing
+## 3. Contact
 
-We're looking forward to your contributions via pull requests.
+If you have further questions, want to provide feedback or you are running into issues, feel free to [open a Github issue]().
 
-### 9.1 Code of Conduct
+### 3.1 E-Mail
+Feel free to get in touch via email to [mobilecentersdk@microsoft.com](mailto:mobilecentersdk@microsoft.com).
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact opencode@microsoft.com with any additional questions or comments.
+### 3.2 Intercom
+Log in to the [Mobile Center portal](https://mobile.azure.com) and use the blue intercom button on the bottom right to start a conversation through our intercom.
 
-### 9.2 Contributor License
-
-You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
+### 3.2 Twitter
+We're on Twitter as [@mobilecenter](https://www.twitter.com/mobilecenter).
 
-## 10. Contact
-If you have further questions or are running into trouble that cannot be resolved by any of the steps here, feel free to open a Github issue here or contact us at mobilecentersdk@microsoft.com.
+### 3.3 Stackoverflow
```

Comment:
@elamalani our readme has it right now and I think it was you who added it. Tell me about it! ;)